python/pyspark/pandas/plot/core.py

```diff
@@ -841,7 +841,7 @@ def barh(self, x=None, y=None, **kwargs):
         elif isinstance(self.data, DataFrame):
             return self(kind="barh", x=x, y=y, **kwargs)
 
-    def box(self, **kwds):
+    def box(self, precision=0.01, **kwds):
```

Comment:
on second thought, I think we'd better still hide `precision` in `**kwds`, because it is a spark-sepcific implementation details.

we can document it in under the `**kwds` docstring.

Reply:
That makes sense! Created https://issues.apache.org/jira/browse/SPARK-50001 for that.